plaso/parsers/aws_elb_access.py

```diff
@@ -0,0 +1,312 @@
+# -*- coding: utf-8 -*-
+"""Parser for AWS ELB access logs.
+
+This parser is based on the log format documented at
+https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html
+"""
+import pyparsing
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.lib import errors
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+class AWSELBEventData(events.EventData):
+  """AWS Elastic Load Balancer access log event data
+
+  Attributes:
+    type (str): The type of request or connection.
```

Comment:
since type is a Python keyword use request_type or equivalent

Reply:
Fixed in last commit